pkg/controller/apmserver/pod.go

```diff
@@ -39,6 +40,11 @@ var (
 			corev1.ResourceMemory: DefaultMemoryLimits,
 		},
 	}
+
+	// DefaultAnnotations are the default annotations for the apmserver pods
+	DefaultAnnotations = map[string]string{
+		annotation.FilebeatModuleAnnotation: "apmserver",
```

Comment:
I left a message on https://github.com/elastic/cloud-on-k8s/issues/3721 because I'm not sure that this module is already available, see https://github.com/elastic/beats/issues/15054